### PR TITLE
Redirect to legacy versions and remove from current documentation

### PR DESCRIPTION
**Pull request by JeremyLikness**
## Summary

Remove from TOC and redirect to legacy. This addresses [issue 2770](https://github.com/MicrosoftDocs/feedback/issues/2770) in the [MicrosoftDocs repo](https://github.com/MicrosoftDocs/).

### CODE REVIEW

**Comment by JeremyLikness** [comment]
@mairaw and @stevestein this is ready for review - the writing section is forwarded to legacy.

File: docs/framework/data/adonet/ef/toc.yml
Comment (adegeo) [inline docs/framework/data/adonet/ef/toc.yml:15]: This indentation is peculiar. The rest of the files in this repo (and others that I've seen) do not indent unnecessarily. I would fix all the indentation in this file to be more inline with the other toc files. While it feels unnatural to NOT indent, you don't need to indent child items. This is perfectly acceptable (and seems to be the preferred way of indentation) where the `-` character of a new items is aligned to the left-column of the parent.

```yml
- name: Forms
  items:
  - name: Overview
    href: file.md
  - name: Event handlers
    href: forms/events.md
  - name: Automatic scaling
    href: forms/autoscale.md
  - name: Common tasks
    items:
    - name: Add a form
      href: forms/how-to-add.md
```

Comment (JeremyLikness) [reply docs/framework/data/adonet/ef/toc.yml:15]: @adegeo @tdykstra is there a standard formatter? without the indentation, it wasn't rendering the TOC correctly. I just used VS Code "format document" then it rendered fine. What's the recommended way to ensure the file formats correctly without manually indenting each line?

Comment (adegeo) [reply docs/framework/data/adonet/ef/toc.yml:15]: My guess as to why it didn't render correctly was just that the system probably inferred spacing based on the other entries, so you can't mix/match indentation, you have to go with one unit of measurement. These are also just my observations 😄 probably any indentation works as long as the file is uniform.

I just like consistency, especially throughout the same repo. Here is the FSharp file for example: https://github.com/dotnet/docs/blob/master/docs/fsharp/toc.yml

Comment (tdykstra) [reply docs/framework/data/adonet/ef/toc.yml:15]: > What's the recommended way to ensure the file formats correctly without manually indenting each line?

Manually indenting is the only way I know to get indentation consistent with the other toc.yml files in the docs repo.

Comment (JeremyLikness) [reply docs/framework/data/adonet/ef/toc.yml]: Should be good.

**Comment by JeremyLikness** [comment]
@tdykstra @adegeo @mairaw I believe the formatting issues are addressed so should be good to go pending any other feedback.

**Comment by adegeo** [comment]
You originally tagged @stevestein do you want them to specifically review this?

**Comment by JeremyLikness** [comment]
@adegeo  No, I believe they were default tagged when I created it. Am I supposed to hit the "merge" button? Given the staleness, should I resync locally a publish before merging or should it be good?

**Comment by tdykstra** [comment]
@JeremyLikness OK to merge. If there were any conflicts GitHub would disable the Merge button.

**Review by tdykstra — APPROVED** [review]

**Pull request merged**
